game/index.html

```diff
@@ -1,6 +1,14 @@
th-child(2):before{-webkit-animation-delay:0s,-1.125s;-moz-animation-delay:0s,-1.125s;-o-animation-delay:0s,-1.125s;animation-delay:0s,-1.125s}.la-ball-atom>div:nth-child(3){-webkit-transform:rotate(120deg);-moz-transform:rotate(120deg);-ms-transform:rotate(120deg);-o-transform:rotate(120deg);transform:rotate(120deg);-webkit-animation-delay:-.25s;-moz-animation-delay:-.25s;-o-animation-delay:-.25s;animation-delay:-.25s}.la-ball-atom>div:nth-child(3):before{-webkit-animation-delay:-1s,-0.75s;-moz-animation-delay:-1s,-0.75s;-o-animation-delay:-1s,-0.75s;animation-delay:-1s,-0.75s}.la-ball-atom>div:nth-child(4){-webkit-transform:rotate(240deg);-moz-transform:rotate(240deg);-ms-transform:rotate(240deg);-o-transform:rotate(240deg);transform:rotate(240deg);-webkit-animation-delay:.25s;-moz-animation-delay:.25s;-o-animation-delay:.25s;animation-delay:.25s}.la-ball-atom>div:nth-child(4):before{-webkit-animation-delay:-0.5s,-0.125s;-moz-animation-delay:-0.5s,-0.125s;-o-animation-delay:-0.5s,-0.125s;animation-delay:-0.5s,-0.125s}.la-ball-atom.la-sm{width:16px;height:16px}.la-ball-atom.la-sm>div:not(:nth-child(1)):before{width:4px;height:4px;margin-top:-2px;margin-left:-2px}.la-ball-atom.la-2x{width:64px;height:64px}.la-ball-atom.la-2x>div:not(:nth-child(1)):before{width:20px;height:20px;margin-top:-10px;margin-left:-10px}.la-ball-atom.la-3x{width:96px;height:96px}.la-ball-atom.la-3x>div:not(:nth-child(1)):before{width:30px;height:30px;margin-top:-15px;margin-left:-15px}@-webkit-keyframes ball-atom-position{50%{top:100%;left:100%}}@-moz-keyframes ball-atom-position{50%{top:100%;left:100%}}@-o-keyframes ball-atom-position{50%{top:100%;left:100%}}@keyframes ball-atom-position{50%{top:100%;left:100%}}@-webkit-keyframes ball-atom-size{50%{-webkit-transform:scale(0.5, 0.5);transform:scale(0.5, 0.5)}}@-moz-keyframes ball-atom-size{50%{-moz-transform:scale(0.5, 0.5);transform:scale(0.5, 0.5)}}@-o-keyframes ball-atom-size{50%{-o-transform:scale(0.5, 0.5);transform:scale(0.5, 0.5)}}@keyframes ball-atom-size{50%{-webkit-transform:scale(0.5, 0.5);-moz-transform:scale(0.5, 0.5);-o-transform:scale(0.5, 0.5);transform:scale(0.5, 0.5)}}@-webkit-keyframes ball-atom-zindex{50%{z-index:10}}@-moz-keyframes ball-atom-zindex{50%{z-index:10}}@-o-keyframes ball-atom-zindex{50%{z-index:10}}@keyframes ball-atom-zindex{50%{z-index:10}}@-webkit-keyframes ball-atom-shrink{50%{-webkit-transform:translate(-50%, -50%) scale(0.8, 0.8);transform:translate(-50%, -50%) scale(0.8, 0.8)}}@-moz-keyframes ball-atom-shrink{50%{-moz-transform:translate(-50%, -50%) scale(0.8, 0.8);transform:translate(-50%, -50%) scale(0.8, 0.8)}}@-o-keyframes ball-atom-shrink{50%{-o-transform:translate(-50%, -50%) scale(0.8, 0.8);transform:translate(-50%, -50%) scale(0.8, 0.8)}}@keyframes ball-atom-shrink{50%{-webkit-transform:translate(-50%, -50%) scale(0.8, 0.8);-moz-transform:translate(-50%, -50%) scale(0.8, 0.8);-o-transform:translate(-50%, -50%) scale(0.8, 0.8);transform:translate(-50%, -50%) scale(0.8, 0.8)}}
```

Comment:
I had to try a few different animations - it was fun finding out which things work while the main thread is blocked.

Reply:
This is some crazy CSS magic. Nice, I like it!

Reply:
To be clear, I stole it wholesale (MIT License in-line).